stable/sysdig/README.md

```diff
@@ -146,3 +172,46 @@ $ cd stable/sysdig
 $ ./scripts/appchecks2helm appChecks/solr.py appChecks/traefik.py appChecks/nats.py > custom-app-checks.yaml
 $ helm install --name sysdig -f custom-app-checks.yaml stable/sysdig
 ```
+
+## Using this chart with AWS Marketplace
+
+This is an use case similar to pull images from a private registry. You need to
+get the authorization token for the image registry:
+
+```bash
+aws ecr --region=us-east-1 get-authorization-token --output text --query authorizationData[].authorizationToken | base64 -d | cut -d: -f2
+```
+
+And use it for creating the ConfigMap. Please, don't forget to replace
+TOKEN and EMAIL with your own values:
+
+```bash
+kubectl create secret docker-registry aws-marketplace-credentials \
+ --docker-server=217273820646.dkr.ecr.us-east-1.amazonaws.com \
+ --docker-username=AWS \
+ --docker-password="TOKEN" \
+ --docker-email="EMAIL"
+j
+```
+
+And the create a YAML file for passing it to Helm chart deployment, for example
```

Comment:
Next you need to create a values YAML file to pass the specific ECS registry configuration (you will find these values when you activate the software from the AWS Marketplace):